tests/unit/repos/test_factory.py

```diff
@@ -0,0 +1,53 @@
+from unittest.mock import Mock
+
+import pytest
+from yarl import URL
+
+from eligibility_signposting_api.repos.factory import dynamodb_resource_factory, s3_service_factory
+
+
+@pytest.fixture
+def mock_session():
```

Comment:
```suggestion
def mock_session() -> Session:
```

You'll also want `from boto3 import Session`